mep/people/urls.py

```diff
@@ -5,6 +5,7 @@
 
 urlpatterns = [
     url(r'^members/$', views.MembersList.as_view(), name='members-list'),
+    url(r'^members/(?P<slug>[\s\w\.\'-]+)/$', views.MemberDetail.as_view(), name='member-detail'),
```

Comment:
Why `\s`? We don't want any whitespace in our URLs

Reply:
At least one mepid had whitespace ([this person](https://test-mep.cdh.princeton.edu/admin/people/person/3608/change/?_changelist_filters=q%3DLa%26p%3D1%26o%3D-10.3)), so I was handling that case. seems like a mistake, since it's not consistent with the other mep IDs?

Reply:
Yeah, that's a mistake in the data. Not sure how we should clean up, but I think ok to ignore it for now - if we decide to use mep ids for urls, we'll need to add those that are missing and standardize the ones that are not (more than this one, I'm sure), and if we decide to create a new slug the mep id inconsistency won't matter.